tedbottompicker/src/main/java/gun0912/tedbottompicker/adapter/GalleryAdapter.java

```diff
@@ -193,31 +188,24 @@ public void onBindViewHolder(final GalleryViewHolder holder, final int position)
                     e.printStackTrace();
                 }
             }
-            if (builder.imageProvider == null) {
```

Comment:
For our use this is fine, but for possibly open sourcing this, removing the imageProvider code here will mean that anyone using the library will be required to use the same specific version of glide.  With the provider a user can use any image library or version they want.

Reply:
Yeah, this version is definitely tailored for Rodio.